articles/openshift/oshift-ref-kibana-dashboards.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Explanation of the custom dashboards we provide with the aggregated logging kibana | UKCloud Ltd
```

Comment:
I'd simplify this title a little bit:
Custom dashboards provided with aggregated logging kibana